bcda/models/models.go

```diff
@@ -168,6 +169,36 @@ func (job *Job) GetEnqueJobs(resourceTypes []string, since string, newBeneficiar
 	return enqueJobs, nil
 }
 
+// Sets the priority for the job where the lower the number the higher the priority in the queue.
+// Prioirity is based on the request parameters that the job is executing on.
+func setJobPriority(acoID string, resourceType string, sinceParam bool) int {
+	var priority int
+	if isSyntheticACO(acoID) {
+		priority = 10 // priority level for jobs for sythetic ACOs that are used for smoke testing
+	} else if resourceType == "Patient" || resourceType == "Coverage" {
+		priority = 20 // priority level for jobs that only request smaller resources
+	} else if sinceParam {
+		priority = 30 // priority level for jobs that only request data for a limited timeframe
+	} else {
+		priority = 100 // default priority level for jobs
+	}
+	return priority
+}
+
+// Checks to see if an ACO is synthetic based on a list of sythetic ACOS provided by an
+// environment variable.
+func isSyntheticACO(acoID string) bool {
```

Comment:
Instead of using the term "synthetic ACO", can we instead replace with "high priority ACO"?  That way, its a bit more flexible - we can add other ACOs which aren't synthetic in the list.

Reply:
Should `isSyntheticACO` be renamed to `isPriorityACO`?  
I think we shouldn't have the term "synthetic ACO" in any of this - instead, use "priority ACO".